app/controllers/api/projects.php

```diff
@@ -1052,7 +1052,10 @@
             ->setAttribute('url', $url)
             ->setAttribute('security', $security)
             ->setAttribute('httpUser', $httpUser)
-            ->setAttribute('httpPass', $httpPass);
+            ->setAttribute('httpPass', $httpPass)
+            ->setAttribute('errors', 0)
+            ->setAttribute('status', true)
+            ->setAttribute('logs', '');
```

Comment:
I think we should add `status` and `logs` into response model of webhook, so we can show it in Console.

In console there should be warning icon (or possibly error one) explaining that what's going on. Showing logs can also help developer debug it.
100% we should also provide information on what to do next to resolve this. For example deleting a webhook and adding it back again.

I also think it should auto-re-enable when you do any change to webhook (I think updating is allowed).

Also I think there should be button to re-enable it. You might be having problem with external server that you now fixed, and simply want to re-enable webhook.

Reply:
The `attempts` count is reset everytime the `update` endpoint is called for Webhooks.

Reply:
Also, who should I assign the frontend changes to?